shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -278,10 +365,17 @@ - (void)loadView {
     }
     flutterView = [[FlutterView alloc] initWithMainContext:mainContext reshapeListener:self];
   }
-  self.view = flutterView;
+  FlutterNativeTextFieldHidder* nativeTextFieldHidder =
+      [[FlutterNativeTextFieldHidder alloc] initWithFlutterView:flutterView];
+  [nativeTextFieldHidder addSubview:flutterView];
+  self.view = nativeTextFieldHidder;
+  self.flutterView = flutterView;
 }
 
 - (void)viewDidLoad {
+  FlutterAppDelegate* appDelegate = (FlutterAppDelegate*)[NSApp delegate];
```

Comment:
Shouldn't we be checking if there are accessibility elements to add to the window here?

Also similar comment to above: is it really safe to assume we'll have a `FlutterAppDelegate` as the app delegate?